docs/ports.md

```diff
@@ -0,0 +1,8 @@
+# Default Ports
```

Comment:
We should probably move these to the book and make these suggested across providers

Reply:
ok sounds good, will open a PR for the book and remove this here

Reply:
@vincepri done here.

Reply:
PR here: https://github.com/kubernetes-sigs/cluster-api/pull/2159